src/app/editor.js

```diff
@@ -2,10 +2,32 @@
 
 var EventManager = require('../lib/eventManager')
 
+var csjs = require('csjs-inject')
 var ace = require('brace')
 var Range = ace.acequire('ace/range').Range
 require('../mode-solidity.js')
 
+var css = csjs`
```

Comment:
Really? But LESS is a separate language and doesn't shield you from css cascading.
That sounds a bit sad, but if that decision is fix i can put it back into the css file